### ISSUE TITLE
Can not use gruf in a non-rails ruby application

### ISSUE THREAD

**Issue by XinchenShen** [bigcommerce/gruf#34]
gruf is not rails specific, but I hit the following exception when I try to use gruf in a non-rails application. It complains that it can not load rails.

```bash
$ bundle exec gruf
bundler: failed to load command: gruf (/Users/xinchen.shen/.rbenv/versions/2.4.1/bin/gruf)
LoadError: cannot load such file -- rails
  /Users/xinchen.shen/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gruf-2.0.2/bin/gruf:20:in `require'
  /Users/xinchen.shen/.rbenv/versions/2.4.1/lib/ruby/gems/2.4.0/gems/gruf-2.0.2/bin/gruf:20:in `<top (required)>'
  /Users/xinchen.shen/.rbenv/versions/2.4.1/bin/gruf:22:in `load'
  /Users/xinchen.shen/.rbenv/versions/2.4.1/bin/gruf:22:in `<top (required)>'
```

**Comment by Parad0X** [comment]
The problem is that `LoadError` is not a subclass of `StandardError` so it can't be rescued with `rescue nil`

```ruby
LoadError.new.is_a?(StandardError)
# => false
```

**Comment by splittingred** [comment]
Fixed by #35

**Comment by splittingred** [comment]
2.0.3 released to enable this fix: https://rubygems.org/gems/gruf/versions/2.0.3
